item/historical/scripts/T000.ipynb

```diff
@@ -364,9 +364,667 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 4,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
```

Comment:
The description of this PR only mentions "merging scripts T007 to S012". What are these changes to T000? We need to have an intelligible description of all changes in a PR, in order to:
- not force reviewers to read the code line-by-line and guess at the intent and effect of changes, and
- allow others, later, to understand why those changes were made.